cpu/atmega1281/include/cpu_conf.h

```diff
@@ -29,17 +29,22 @@ extern "C" {
  * @name Kernel configuration
  *
  * Since printf seems to get memory allocated by the linker/avr-libc the stack
- * size tested sucessfully even with pretty small stacks.k
+ * size tested sucessfully even with pretty small stacks.
  * @{
  */
 #define THREAD_EXTRA_STACKSIZE_PRINTF    (128)
 
 #ifndef THREAD_STACKSIZE_DEFAULT
-#define THREAD_STACKSIZE_DEFAULT   (256)
+#   define THREAD_STACKSIZE_DEFAULT   (256)
```

Comment:
That's not what I meant... in another PR @OlegHahm asked me to remove those kind of indentations in pre-processor directives altogether. So

```C
#ifndef FOO
#define FOO
#endif
```

instead of


```C
#ifndef FOO
#   define FOO
#endif
```

That the `#` needs to be at the beginning of the line everyone knows as soon as they start compiling ;-)